napari/layers/intensity_mixin.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, *args, **kwargs) -> None:
         self._contrast_limits_range = [None, None]
         self._auto_contrast_source = 'slice'
         self._keep_auto_contrast = False
+        self._custom_colormaps = {}
```

Comment:
Note that both classes that use the intensity mixin (`_ImageBase` and `Surface`) define `_colormaps` as a class variable. Not directly related to this PR, but we should just define that class variable in this mixin class instead, given this is the only place we refer to it.

Reply:
Probably good to unify here as well, yeah.